src/main/java/com/feywild/feywild/quest/player/QuestData.java

```diff
@@ -126,6 +126,10 @@ public Alignment getAlignment() {
         return this.alignment;
     }
 
+    public boolean checkReputation(Alignment alignment, int min_reputation) {
+        return this.alignment == alignment && this.reputation > min_reputation;
+    }
+
```

Comment:
You should then also use that method. Also it should return true if the reputations are equal.